README.md

```diff
@@ -4,22 +4,123 @@
 [![Discord](https://img.shields.io/discord/1074074312421683250?color=%237289da&label=discord)](https://discord.gg/Qcqf9R27BR)
 
 # avaje-logback-encoder
-logback encoder that uses avaje-jsonb to log events as json
+Logback encoder that log events as json (similar to Logstash).
 
-## Usage
+Example:
+```json
+{"timestamp":"2025-01-10T14:47:42.313+13:00","level":"INFO","logger":"org.example.Foo","message":"Hi","thread":"main"}
+```
+
+Example with component and environment:
+```json
+{"component":"my-component","env":"DEV","timestamp":"2025-01-10T14:47:42.313+13:00","level":"INFO","logger":"org.example.Foo","message":"Hi","thread":"main"}
+```
+
+## Fields
+#### Standard Fields
+- `timestamp`
+- `level`
+- `logger`
+- `message`
+- `thread`
+- `stacktrace`
+
+#### MDC Fields
+MDC key/values are included in logged events.
+
+#### Custom Fields
+Extra Custom fields can be declared in JSON form, these are added to all logged events.
+
+#### Extra recommended Fields
+- `component` - Use to define the "component" (approximately application or a specific component of an application)
+- `env` - Use to define the "environment" such as dev, test, prod etc
+
+These default by reading System Environment variables `COMPONENT` and `ENVIRONMENT` respectively and
+can also be explicitly set via configuration.
+
+
+
+# How to use
+
+#### 1 - Add dependency
+
+For Java 11+ and Logback 1.2.x+ use version `1.0` of the dependency:
+```xml
+<dependency>
+  <groupId>io.avaje</groupId>
+  <artifactId>avaje-logback-encoder</artifactId>
+  <version>1.0</version>
+</dependency>
+```
 
-Add the encoder to your appender
+For Java 8 and Logback 1.1.x use version `1.0-java8` of the dependency.
```

Comment:
Java 8???

Reply:
Needs must - but yes Java 8 and Logback 1.1.11 ( + Glassfish)

Reply:
There is the java8-backport branch where this lives